src/Whoops/Resources/css/whoops.base.css

```diff
@@ -426,3 +426,8 @@ kbd {
     padding: 3px 5px;
     vertical-align: middle;
 }
+
+pre.sf-dump {
+    padding: 0px !important;
```

Comment:
its ugly, but atm the only way to override the styles the symfony/var-dumper renders into the body using `<style>`-tags